backend/src/ml_space_lambda/data_access_objects/dynamo_data_store.py

```diff
@@ -63,9 +67,14 @@ def _query(
         index_name: Optional[str] = None,
         limit: Optional[int] = None,
         page_response: bool = False,
+        scan_index_forward: bool = True,
```

Comment:
I did a quick spot check and I think we are always using named args when calling this but generally adding a new optional parameter before an existing parameter can cause issues.

Reply:
Good point, I'll bump it to the bottom just in case.